### PR TITLE
Auth backend saml clock discrepancies

### LINKED ISSUE #6608: Allow to configure SAML auth acceptedClockSkewMs
SAML authentication can be configure in app-config.yaml file. Some options are available such as entryPoint, issuer or other.
acceptedClockSkewMs option is not available.

## Feature Suggestion

Allow to configure SAML auth acceptedClockSkewMs

## Possible Implementation

I already tried an implementation.

in plugins/auth-backend/config.d.ts b/plugins/auth-backend/config.d.ts
add `acceptedClockSkewMs: number;` to Config interface

in plugins/auth-backend/src/providers/saml/provider.ts b/plugins/auth-backend/src/providers/saml/provider.ts
add `acceptedClockSkewMs: config.getOptionalNumber('acceptedClockSkewMs'),` to SAML authentication provider options

## Context

I can start to work on it.

### PR DESCRIPTION
**Pull request by stephane-mori**
## This is a pull request about SAML configuration

It allow to configure acceptedClockSkewMs option.
Closes #6608 

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/auth-backend/config.d.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ export interface Config {
         decryptionPvk?: string;
         signatureAlgorithm?: 'sha256' | 'sha512';
         digestAlgorithm?: string;
+        acceptedClockSkewMs: number;
```
Comment (benjdlambert) [inline plugins/auth-backend/config.d.ts:56]: This should be `acceptedClockSkewMs?` to make it optional.

Comment (stephane-mori) [reply plugins/auth-backend/config.d.ts]: I fixed it

File: .changeset/lazy-apples-scream.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': minor
```
[the remark this answers is not in the document]
Comment (stephane-mori) [reply .changeset/lazy-apples-scream.md]: I fixed it

**Comment by stephane-mori** [comment]
I dont understand why this fail : https://github.com/backstage/backstage/actions/runs/1071270059

**Comment by benjdlambert** [comment]
Looks like a swagger dependency release broke the e2e tests however, we've fixed it in #6626 so you should be able to go ahead and rebase from master and the build pass.

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**